examples/platforms/cc1352/arm-none-eabi.cmake

```diff
@@ -28,6 +28,7 @@
 
 set(CMAKE_SYSTEM_NAME              Generic)
 set(CMAKE_SYSTEM_PROCESSOR         ARM)
+set(CMAKE_BUILD_TYPE               Release CACHE STRING "Set build type to release to save memory")
```

Comment:
I think we should let user specify the build type.

Reply:
can we move it into script/cmake-build if  user does not specify CMAKE_BUILD_TYPE?